crates/composition/tests/composition/shareable_basic/federated.graphql

```diff
@@ -1,11 +1,12 @@
 type Customer {
-    id: ID
+    id: ID!
     name: String
     other: Int
     newsletterSubscribed: Boolean
-    subscriptionPlan: Plan
+    subscriptionPlan: Plan!
```

Comment:
I assume all these non-nullable changes are expected?

Reply:
Yes, we weren't writing the bangs and list type wrappers at all before, so all types were written as nullable non-list. Now they have the wrappers.